src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
-export interface Figure {
+// import { type } from "os";
+
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'blue' | 'green' | 'red';
 
+export interface Figure {
+  shape: Shape
+  color: Color
+  getArea(): number
+  radius?: number
+  sides?: number[]
```

Comment:
In Figure interface you should specify only those properties that should be for all figures
Remove radius and sides from this interface and add it to the needed figures